Wikipedia/Code/WMFColumnarCollectionViewLayout.m

```diff
@@ -121,6 +121,7 @@ - (UICollectionViewLayoutInvalidationContext *)invalidationContextForBoundsChang
     WMFCVLInvalidationContext *context = (WMFCVLInvalidationContext *)[super invalidationContextForBoundsChange:newBounds];
     context.boundsDidChange = YES;
     context.newBounds = newBounds;
+    self.info = [self.info copy];
```

Comment:
no - was testing and accidentally pushed after making the other fixes. reverted the revert commit.